src/stream.c

```diff
@@ -239,7 +239,7 @@ void avifRWStreamFinishWrite(avifRWStream * stream)
     }
 }
 
-void avifRWStreamWrite(avifRWStream * stream, const uint8_t * data, size_t size)
+void avifRWStreamWrite(avifRWStream * stream, const void * data, size_t size)
```

Comment:
We do not do pointer arithmetic with the 'data' parameter in this function, so we can just declare it as const void *, like the memcpy() and memcmp() functions.